packages/vscode-extension/package.json

```diff
@@ -291,6 +291,41 @@
           "command": "fx-extension.grantPermission",
           "when": "view == teamsfx-environment && viewItem =~ /environment/",
           "group": "inline@3"
+        },
+        {
+          "command": "fx-extension.openDocumentLink",
+          "when": "view == teamsfx-development && viewItem == fx-extension.create",
```

Comment:
Why not make all these scenarios in one and update 'when' condition?

Reply:
Thanks, updated.